zulip/zulip/__init__.py

```diff
@@ -1554,6 +1554,53 @@ def set_typing_status(self, request: Dict[str, Any]) -> Dict[str, Any]:
             request=request
         )
 
+    def move_topic(
+        self,
+        stream: str,
+        new_stream: str,
+        topic: str,
+        new_topic: Optional[str] = None
```

Comment:
Ideally, this would support the `propagate_mode` option and a message ID threshold.  https://zulipchat.com/api/update-message#parameters has some basic documentation